### PR TITLE
UserAgent: Removes direct version

### LINKED ISSUE #3160: UserAgent: Remove direct version
The direct version should not be included in the .NET v3 user agent. The direct is not done via a nuget reference so it is not necessary. This has caused a lot of confusion for customers and has not been used in any scenarios. 

Current example:
`cosmos-netstandard-sdk/3.26.1|3.27.2|1|X64|Microsoft Windows 10.0.22000|.NET 6.0.4|N|`

The 3.27.1 should be removed. This can also be figured out from the release branch.
Updated:
`cosmos-netstandard-sdk/3.26.1|1|X64|Microsoft Windows 10.0.22000|.NET 6.0.4|N|`

### PR DESCRIPTION
**Pull request by imanvt**
# Pull Request Template

## Description

Remove direct version from user agent to as it is unnecessary. The direct version dll is copied into the nuget folder rather than being referenced as a nuget so customers can not change it.

## Type of change
- New feature (non-breaking change which adds functionality)

## Closing issues
closes #3160

### CODE REVIEW

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/UserAgentTests.cs [hunk 1]
```diff
@@ -93,7 +93,6 @@ public void VerifyUserAgentContent()
             previewFlag = "P";
 #endif
             Assert.AreEqual($"cosmos-netstandard-sdk/{envInfo.ClientVersion}" + previewFlag, values[0]);
-            Assert.AreEqual(envInfo.DirectVersion, values[1]);
             Assert.AreEqual("0", values[2]);
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/UserAgentTests.cs:96]: Doesn't this number have to be updated?

**Review by j82w — APPROVED** [review]

**Comment by sourabh1007** [comment]
Now onwards..what is the right way to check which version of SDK is using which version of Direct package?

**Comment by j82w** [comment]
The user agent still has the SDK version. The direct version can be found by looking it up in the github branches. There is a branch for every release.

**Comment by sourabh1007** [comment]
> The user agent still has the SDK version. The direct version can be found by looking it up in the github branches. There is a branch for every release.

Should we make it part of our changelog? that will make life easier instead of searching for it in the branch

**Comment by j82w** [comment]
It's very rare that we need to know the direct version. I don't think we need to log it in the changelog, and it's not something customer should know or care about.

**Pull request merged**